### ISSUE TITLE
Ethoscope going into emergency mode

### ISSUE THREAD

**Issue by ThHoler** [gilestrolab/ethoscope#145]
Hello, 

Recently our ethoscopes have been having a new issue. Once an experiment is done and we end the acquisition, the ethoscope sometimes disappears entirely from the list of available machines on the Platform Overview page. This happens without us actually shutting the ethoscope down. 

We tried connecting a monitor to the ethoscope using an HDMI cable and this is what we saw :
![20210201_123602](https://user-images.githubusercontent.com/73488650/106457191-e5e42580-648e-11eb-99b7-cb7f9c94833d.jpg)

So far the only solution we have found to that issue was to reset the SD card and flash it again, which is quite bothersome considering it happens somewhat often. What could be the reason for that issue and the solution to it ?

Thanks in advance.

**Comment by ggilestro** [comment]
This is a sign of failure of the SD card. Unfortunately, the failure rate in SD cards is the weakest aspect of the platform at the moment and depends largely on the brand of SD cards one uses. We have had the best results with Samsung cards in the past.

**Comment by ThHoler** [comment]
Thank you for the answer.

We have been using Samsung 64 gb cards. Is there any way to mitigate the risk of SD card failure ? I suppose it isn't possible to use a USB stick to host the software for instance ?

**Comment by ggilestro** [comment]
I have made a change to the code that removes some heavy writing activity on the SD card upon completion of the experiment. Perhaps this was the issue. You may want to try and update your ethoscopes now.

**Comment by ggilestro** [comment]
Closed for lack of activity
